### DISCUSSION TITLE
Project

### DISCUSSION THREAD

**Discussion by Dhruvaggr** [community/community#59737] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am new to GIT Hub, please see my latest project or any project and seek some advice in it.

**Comment by GhostOf0days** [comment]
Seperate your files into folders. For instance, put your source code in a folder called src (/src). Ideally, further separate them depending on their functions (i.e. For a parser, add an additional folder called "parser" for parsing source code). As for images, put them into a folder called images in a folder called public (/public/images). Also, you should use [camelCase](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions) for your variables.

**Reply by GhostOf0days** [reply]
Also, use business logic comments to make code clear to understand to even people unfamiliar with your code base and remove unused code.
